smoketest/twiliosmsstopstart_test.go

```diff
@@ -46,21 +46,20 @@ func TestTwilioSMSStopStart(t *testing.T) {
 	h := harness.NewHarness(t, sql, "ids-to-uuids")
 	defer h.Close()
 
+	dev := h.Twilio(t).Device(h.Phone("1"))
```

Comment:
It might make more sense to name this `d1` for consistency

```suggestion
	d1 := h.Twilio(t).Device(h.Phone("1"))
```